test/integration_test/applicationbackup_test.go

```diff
@@ -266,11 +289,18 @@ func triggerScaleBackupRestoreTest(
 	}
 	// Cleanup
 	for idx, bkp := range appBkps {
-		err := storkops.Instance().DeleteBackupLocation("backuplocation-"+strconv.Itoa(idx), bkp.Namespace)
-		require.NoError(t, err, "Failed to delete  backup location %s in namespace %s: %v", "backuplocation-"+strconv.Itoa(idx), bkp.Namespace, err)
+		// Get backuplocation so it can be used in the invocation of the validateBackupDeletionFromObjectstore method
```

Comment:
This is required because `backup.Status.BackuPath` is populated only after backup is completed.